packages/plugin-conversation/src/interceptor.js

```diff
@@ -24,27 +29,22 @@ export default class ConversationInterceptor extends Interceptor {
    * @returns {Promise<Object>}
    */
   onRequest(options) {
-    if (!options.method || options.method.toUpperCase() === `GET`) {
-      return options;
-    }
-
-    return this.spark.device.isSpecificService(`conversation`, options.service || options.uri)
-      .then((isConversationService) => {
-        if (!isConversationService) {
+    return this.shouldEncryptRequest(options)
+      .then((shouldDecrypt) => {
+        if (!shouldDecrypt) {
           return options;
```

Comment:
this one returns options, while the one below returns Promise.resolve(). is that correct?